### PR TITLE
Bind base directory for fs-events into user job container

### PR DESCRIPTION
**Pull request by koct9i**
Fix tests with brakepoints in CRI environment.

Fix for: https://github.com/ytsaurus/ytsaurus/issues/226

I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

**Comment by gritukan** [comment]
Let's add some simple test with breakpoint

File: yt/yt/tests/library/yt_env_setup.py [hunk 1]
```diff
@@ -819,6 +819,13 @@ def apply_config_patches(cls, configs, ytserver_version, cluster_index):
 
             config["ref_counted_tracker_dump_period"] = 5000
 
+            # FIXME(khlebnikov) quick hack for "with_breakpoint".
+            config["exec_node"].setdefault("root_fs_binds", []).append({
```
Comment (gritukan) [inline yt/yt/tests/library/yt_env_setup.py:823]: What is a non-hack solution here?

**Comment by koct9i** [comment]
> Let's add some simple test with breakpoint

these are simple enough
https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/tests/integration/node/test_jobs.py#L74

File: yt/yt/tests/library/yt_env_setup.py (hunk 1 above)
Comment (gritukan) [reply yt/yt/tests/library/yt_env_setup.py:823]: I'd suggest changing FIXME to TODO with this idea.

**Comment by yalekseev** [comment]
Shipped!

**Pull request closed without merging**